networking/ovn_kubernetes_network_provider/ovn-k-network-policy.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ovn-k-network-policy"]
+= OVN-Kubernetes network policy
+include::_attributes/common-attributes.adoc[]
+:context: ovn-k-network-policy
+
+toc::[]
+
+:FeatureName: The `AdminNetworkPolicy` resource
+include::snippets/technology-preview.adoc[]
+
+Kubernetes offers two features that users can use to enforce network security. One feature that allows users to enforce network policy is the `NetworkPolicy` API that is designed mainly for application developers and namespace tenants to protect their namespaces by creating namespace-scoped policies. For more information, see xref:../../networking/network_policy/about-network-policy.adoc#nw-networkpolicy-about_about-network-policy[About network policy].
+
+The second feature is `AdminNetworkPolicy` which is comprised of two API: the `AdminNetworkPolicy` (ANP) API and the `BaselineAdminNetworkPolicy` (BANP) API. Together ANP and BANP are designed for cluster and network administrators to protect their entire cluster by creating cluster-scoped policies. Cluster administrators can use ANPs and BANPs to enforce non-overridable policies that take precedence over `NetworkPolicy` objects.
```

Comment:
@tssurya and @asood-rh I think we need a more generic name when we talk about "Admin network policy" in the general sense; for example, here as a feature. What do you think of "global network policy"?

Reply:
no we can't put a new name at this stage, I think its important to prefix/suffix the terms feature and API whenever needed but mostly it should be implicitly understood...
perhaps @astoycos has better ideas here since this is important from marketing perspective of our community and project moving forward and we don't want to be stuck with a branding we don't like,